docs/architecture.md

```diff
@@ -4,41 +4,50 @@
   <img src="images/architecture.png" width="800">
 </p>
 
+## Data source
+
+Our data is sourced from a variety of [sources], which we are looking to expand
+on over time.
+
+[sources]: https://github.com/google/osv.dev/tree/master/vulnfeeds
```

Comment:
Let's just link to https://github.com/google/osv.dev#current-data-sources

Reply:
Addressed in be3378d